tests/test_somersault.py

```diff
@@ -305,20 +305,20 @@ def test_free_param_info(self) -> None:
         stdout = StringIO()
         with patch("sys.stdout", stdout):
             request.print_free_parameters_info()
-            expected_output = "forward_soberness_check: uint8\nnum_flips: uint8\n"
+            expected_output = "forward_soberness_check: uint\nnum_flips: uint\n"
```

Comment:
no size info?

Reply:
adding a size lead to quite unexpected results like `float16`. I think the sizes should be removed here anyway, as this function is concerned with the physical types of the parameters and in the physical space, the bit sizes of the internal representation of an object should not matter (only the valid range does).